packages/reporter/src/test/test.tsx

```diff
@@ -71,48 +81,45 @@ class Test extends Component<Props> {
     if (!model.shouldRender) return null
 
     return (
-      <div
-        ref='container'
-        className={cs('runnable-wrapper', { 'is-open': this._shouldBeOpen() })}
-        onClick={this._toggleOpen}
-        style={{ paddingLeft: indent(model.level) }}
+      <Collapsible
+        containerRef={this.containerRef}
+        header={this._header()}
+        headerClass='runnable-wrapper'
```

Comment:
I don't think `runnable-wrapper` should be the header class. It's meant to wrap the entire runnable. You could achieve this by updating the `<Collapsible />` component to accept a `className` prop, and then add that in to the root-level div's className. See my other code comment for further clarification.

That should fix the failing e2e tests. They rely on the `runnable-wrapper` class being on the top-most element of the runnable. It should also reduce the need for the changes to selectors in the `scss` file